packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -291,6 +308,16 @@ export class Relayer {
           at: "InsuredBridgeRelayer#Relayer",
           type: "Relay instantly sent 🚀",
           tx: receipt.transactionHash,
+          chainId: receipt.events.DepositRelayed.returnValues.depositData.chainId,
```

Comment:
nice. we should work on better spreading the `depositData` objects to clean up this syntax. I have some ideas on one of my local branches.